tools/SetupFlow/DevHome.SetupFlow/Views/AppManagementView.xaml

```diff
@@ -78,18 +70,9 @@
                         </AutoSuggestBox>
                     </common:ExperimentControl.DefaultContent>
 
-                    <common:ExperimentControl.ExperimentContent>

```

Comment:
Is the stuff in this file part of what we're removing here?

Reply:
This was part of an experiment testing the effects of changing what was in the AppManagement page's search bar. @AmelBawa-msft, could you explain more about how this was connected to Device Restore?

Reply:
This is not connected to the restore. But it does uses the internal nuget to check if an experiment is on or off. The control is just a wrapper to use it in XAML. If this PR is just for removing the restore experience, then this control doesn't need to  be removed.

Reply:
I've removed the changes to the search bar. I plan to put them in another PR.